src/pty.rs

```diff
@@ -2,6 +2,7 @@ use crate::io::set_non_blocking;
 use crate::tty::{Tty, TtySize};
 use anyhow::{bail, Result};
 use nix::errno::Errno;
+use nix::libc::{SIGALRM, SIGCHLD};
```

Comment:
We already import other signals from `signal_hook::consts` (below), so let's add those there, ok?